sensor_msgs/Image image
float32[100] scores
float32[100] classes
uint16 num_detections
Box[100] boxes
string[100] track_list
